src/sparseml/keras/utils/pytorch2keras.py

```diff
@@ -0,0 +1,347 @@
+import re
```

Comment:
Finally let's add some docs at the top to go over what the script is for and give examples for the command to run with the args mentioned in another diff